FlightDynamics.m

```diff
@@ -1,81 +1,93 @@
+%% Rocket Design Parameters
+Rocket.mass_empty = 15.2171;              %unloaded mass of rocket (kg)
+Rocket.mass_propellant = 2.066;           %mass of propellant (kg)
+Rocket.mass_aft = 4.61;                   %mass of aft section (kg)
+Rocket.mass_payload = 2.0;                %mass of payload (kg)
+Rocket.main_altitude = 198;               %main parachute deployment altitude (m)
+
+Rocket.payload_altitude = 244;            %payload deployment altitude (m)
+Rocket.max_acceleration = 0;              %initialize max acceleration (m/s^2)
+
+Rocket.len = 3.5;                         %rocket length (m)
+Rocket.dia = 0.14;                        %fuselage diameter (m)
+Rocket.s = 0.127;                         %fin half-span (m)
+Rocket.cr = 0.381;                        %fin root chord (m)
+Rocket.ct = 0;                            %fin tip chord (m)
+Rocket.ft = 0.00318;                      %fin thickness (m)
+Rocket.dia_motor = 0.075;                 %motor diameter (m)
+Rocket.dia_drogue = 0.61;                 %drogue parachute diameter (m)
+Rocket.dia_main = 2.13;                   %main parachute diameter (m)
+Rocket.cg = 2.15;                         %center of gravity from nose (m)
+Rocket.cp = 2.59;                         %center of pressure from nose (m)
+
+Rocket.I = 3489;                          %total impuse (N*s)
+Rocket.C_D_drogue = 1.16;                 %drogue parachute drag coefficient
+Rocket.C_D_main = 2.20;                   %main parachute drag coefficient
+Rocket.R_s = 100E-5;                      %surface roughness (m)
+
+Rocket.rail_length = 12;                  %launch rail length (ft)
+Rocket.launch_angle = (4)*pi/180;         %launch angle from vertical (rad)
+
+%Load motor time (s) and thrust (N) data
+Rocket.time_data = csvread('Simulation Thrust.csv',0,0,[0 0 97 0]);     %Time (s)
+Rocket.thrust_data = csvread('Simulation Thrust.csv',0,1,[0 1 97 1]);   %Thrust (N)
 
 %Calculate static stability margin (cal)
-ss = (cp - cg)/d;
+ss = (Rocket.cp-Rocket.cg)/Rocket.dia;
+
+%% Atmospheric Parameters
+Atmos.g = 9.81;                         %gravitational acceleration (m/s^2)
+Atmos.gamma = 1.4;                      %ratio of specific heats
+Atmos.R = 287;                          %gas constant (J/kg*K)
+Atmos.T_ground = 292;                   %temperature on launch pad (K)
+Atmos.lapse_rate = 0.0065;              %troposphere lapse rate (K/m)
+Atmos.turbulence_intensity = 1;         %turbulence intensity (1-light,2-moderate,3-severe)
+Atmos.mu = 18.07E-6;                    %dynamic viscosity (Pa*s)
+Atmos.rho = 1.225;                      %air density (kg/m^3)
 
 % Initialize upper and lower wind values for bandwidth
-windupper = 5; 
-windlower = -5;
-
-% Initialize turbulence intensity
-% Choose from: 
-% light = 1
-% moderate = 2
-% severe = 3
-turbulence_intensity = 1;
+Atmos.windspeed = 5.8115;   %Average windspeed (m/s)( (+) for into the wind, (-) for against the wind) 

```

Comment:
Was previously left at 5,5,-5 which doesnt really make sense.
Asked Wiler and it was a left over from older code. This implementation is more accurate.